resources/assets/sass/components/partials.scss

```diff
@@ -0,0 +1,31 @@
+.alert {
+    padding: 15px;
+    border-radius: 5px;
+    color: $colors-light-background;
+}
+
+.alert--danger {
```

Comment:
`.alert--danger` can only be used in combination with `.alert`, so the CSS selector would be `.alert.alert--danger`.